src/utils/AccessibleButton.ts

```diff
@@ -63,6 +64,11 @@ export class AccessibleButton {
     return this;
   }
 
+  public withKeyboardAction(key: KEYBOARD, action: (e: Event) => void) {
```

Comment:
Since it's not used anywhere in the codebase for now, I would remove it.

Reply:
I reverted this change, thanks.